test/functional/isVisible.js

```diff
@@ -0,0 +1,58 @@
+const { assert } = require('chai');
+const firefox = require('selenium-webdriver/firefox');
+const webdriver = require('selenium-webdriver');
+const { ancestors, isDomElement, isVisible, toDomElement } = require('../../utilsForFrontend');
+
+const { until, By } = webdriver;
```

Comment:
Any reason you chose not to do just `const {Builder, By, until} = require('selenium-webdriver');` and save a line? Did you especially like the explicitness of `webdriver.Builder` below?